### ISSUE TITLE
Documentation has syntax error

### ISSUE THREAD

**Issue by josephys** [lhorie/mithril.js#1283]
Page http://mithril.js.org/mithril.html has syntax error:
### Actual:

In Shorthand syntax:

```
var MyComponent = {
...
}

m.render(document.body, [
    //the two lines below are equivalent
    m(component, {data: "world"}),
    m.component(component, {data: "world"})
])


```
### Expected:

```
var MyComponent = {
...
}

m.render(document.body, [
    //the two lines below are equivalent
    m(MyComponent, {data: "world"}),
    m.component(MyComponent, {data: "world"})
])
```

**Comment by sanspace** [comment]
Seems this have been fixed by the below commit

https://github.com/lhorie/mithril.js/commit/6ffeeaa19d5590d34082c24706a42e4167cb0a15

However, the site is still not updated. Perhaps, a rebuild required!?

**Comment by isiahmeadows** [comment]
@sanspace There's other similar issues that have also been caught, but not updated on the main site. IIRC @lhorie rebuilds the live site on each release (major/minor/patch), so it might be waiting for the next train (which is admittedly overdue IMHO).

**Comment by sanspace** [comment]
So, we are waiting for a train and we do know where it would take us!! :laughing:

**Comment by tivac** [comment]
Fixed w/ 1.0 and new site
